js/components/ImportIndicatorsPopover.js

```diff
@@ -77,7 +77,7 @@ export class ImportIndicatorsButton extends React.Component {
     // Method to store the current popover view and valid/invalid row counts if available
     setStoredView = (view) => {
         this.setState({
-            storedView: view
+            storedView: view.view !== 3 ? view : {view: 0}
```

Comment:
I update the variable name to hopefully make it more clear. I have the parameters being passed in as the object `currentView` now which has the property `view`. So then `storedView` will be a an object, set as either the currentView passed in or the initial `{view: 0}` object. If the `currentView`'s view is 3, which is the index for the Successful upload view, don't store that view but instead revert back to the initial view. This is to prevent the scenario you mentioned when the popover opened back up to the Successful view after the import process was completed.

Reply:
Ok, that helps.